source/src/index.html

```diff
@@ -869,7 +869,7 @@ <h1 id="header-name">Pomodoro Timer</h1>
             <nav id="header-buttons">
                 <ul>
                     <li title="Focus">
-                        <button class="top-buttons" id="focus-button">
+                        <button id="focus-button">
```

Comment:
The top-buttons class cannot be removed since the opacity is set by this class. 
```suggestion
                        <button class="top-buttons" id="focus-button">
```

Reply:
I modified it such that it uses the id to set opacity

> The top-buttons class cannot be removed since the opacity is set by this class.